tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/TestSymbStateRewriterSet.scala

```diff
@@ -813,6 +814,54 @@ class TestSymbStateRewriterSet extends RewriterBase with TestingPredefs {
     }
   }
 
+  // Regression for the issue 365: https://github.com/informalsystems/apalache/issues/365
+  // This test goes throught without a need for a fix.
```

Comment:
```suggestion
  // This test goes through without a need for a fix.
```

Reply:
I don't think this comment is going to add value for any subsequent readers. Seems only relevant for the review, right?